barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -378,23 +377,17 @@ void client_ivc_prove_output_all_msgpack(const std::string& bytecodePath,
 
     // Write the proof and verification keys into the working directory in  'binary' format (in practice it seems this
     // directory is passed by bb.js)
-    std::string vkPath = outputDir + "/mega_vk"; // the vk of the last circuit in the stack
+    std::string vkPath = outputDir + "/client_ivc_vk"; // the vk of the last circuit in the stack
```

Comment:
the comment is stale

Reply:
Made the change in my API PR that builds off this one